cmd/status-cli/README.md

```diff
@@ -0,0 +1,17 @@
+# Status CLI
+
+The Status CLI is a command line interface for interacting with the Status messenging process. It is a tool for developers and QAs to test the communication workflow without running Status desktop and mobile app.
+
+## Features
+
+- Create a new account
+- Send and receive contact request
+- DM between contacts
+
+## Run
+
+```bash
+go run main.go dm > log
```

Comment:
Do we really need to suggest `> log`? This is basic console interaction. 
I'd expect the default way to print result to console.
```suggestion
go run main.go dm
```